src/manager.cpp

```diff
@@ -143,9 +143,8 @@ bool Manager::parseOpdsDom(const pugi::xml_document& doc, const std::string& url
     m_totalBooks = strtoull(libraryNode.child("totalResults").child_value(), 0, 0);
     m_startIndex = strtoull(libraryNode.child("startIndex").child_value(), 0, 0);
     m_itemsPerPage = strtoull(libraryNode.child("itemsPerPage").child_value(), 0, 0);
-    m_hasSearchResult = true;
   } catch(...) {
-    m_hasSearchResult = false;
+    return false;
```

Comment:
The `totalResults`/`startIndex`/`itemsPerPage` nodes are present when parsing a OPDS which is a search result. If parsing a root opds stream, they are not present.
So we must not stop the parsing of the opds stream if they are not present.